numba_cuda/numba/cuda/tests/cudadrv/test_module_callbacks.py

```diff
@@ -185,7 +185,7 @@ def setUp(self):
 
         def set_forty_two(handle):
             # Initialize 42 to global variable `num`
-            res, dptr, size = cuModuleGetGlobal(
+            res, dptr, size = cuLibraryGetGlobal(
```

Comment:
We'll probably have to update the callback signature in the documentation as well:
https://nvidia.github.io/numba-cuda/user/cuda_ffi.html#linking-and-calling-functions